src/lando/main/scm/abstract_scm.py

```diff
@@ -147,7 +147,7 @@ def prepare_repo(self, pull_path: str):
             self.clone(pull_path)
         else:
             with self.for_pull():
-                logger.info(f"Updating {self} from pull path.")
+                logger.info(f"Updating {self.path} from pull path.")
```

Comment:
I think it would be good to update `AbstractSCM.__str__` to show something more useful as well.